mumlib/src/config.rs

```diff
@@ -20,6 +27,10 @@ struct TOMLConfig {
 pub struct Config {
     pub audio: AudioConfig,
     pub servers: Vec<ServerConfig>,
+    /// Whether we allow connecting to servers with invalid server certificates.
+    ///
+    /// None implies false but we can show a better message to the user.
+    pub allow_invalid_server_cert: Option<bool>,
```

Comment:
Since this field is documented, it might be worthwhile to document the rest of the struct too.

Reply:
```rust
    /// Audio configuration.
    pub audio: AudioConfig,
```
I don't really think this is worth it, unless you have ideas for something actually worthwhile to write.

Reply:
Maybe just a doc-comment on the whole struct then?